algos/src/sample_rate_conversion.rst

```diff
@@ -219,21 +218,21 @@ their conversion fraction and these bandwidths.
 		cnv.rp_tot = 0.1; % Max +/- passband ripple allowed, used in test script only
 		cnv.gain = -1; % Gain in decibels at 0 Hz
 
-The next plots show the difference between firpm and kaiser SRC filter
-characteristic. In equiripple the passband and stopband are just at
+The next plots show the difference between the firpm and the kaiser SRC
+filter characteristic. In equiripple, the passband and stopband are just at
 the allowed limit across the pass and stopband. Equiripple design is
-selected with option cnv.design set to ‘firpm’. However in Octave it
-fails in many conversion due to apparent bug in remez() function. In
-Matlab function firpm() is used and it can be used up to about 2000
+selected with the option cnv.design set to ‘firpm’. However, in Octave it
+fails in many conversions due to an apparent bug in the remez() function. In
+Matlab, the function firpm() is used and it can be used for up to about 2000
 order filters.
 
-The cnv.design set to ‘kaiser’ is robust choise for all conversions
+The cnv.design set to ‘kaiser’ is a robust choice for all conversions
```

Comment:
@singalsu Please review this paragraph, beginning at line 229.

Reply:
It's OK